### PR TITLE
doc: Update Configuration and Development Overview

### PR DESCRIPTION
**Pull request by amyreye**
- Align with new Configurator design, and use of Configurator instead of editing predefined XMLs

Signed-off-by: Reyes, Amy <amy.reyes@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

File: doc/getting-started/overview_dev.rst [hunk 1]
```diff
@@ -82,13 +83,12 @@ that ACRN offers:
...
 
-* **Pre-launched User VMs**: These VMs are automatically launched at boot time
-  by the hypervisor. They run independently of other VMs and own dedicated
-  hardware resources, such as a CPU core, memory, and I/O devices. Other VMs,
-  including the Service VM, may not even be aware of a pre-launched VM's
-  existence. The configuration of pre-launched VMs is static and must be defined
-  at build time. They are well-suited for safety-critical applications and where
-  very strict isolation, including from the Service VM, is desirable.
+* **Pre-launched User VMs**: These VMs run independently of other VMs and own
+  dedicated hardware resources, such as a CPU core, memory, and I/O devices.
```
Comment (dbkinder) [inline doc/getting-started/overview_dev.rst:87]:
```suggestion
  dedicated hardware resources, such as a CPU cores, memory, and I/O devices.
```

File: doc/getting-started/overview_dev.rst [hunk 2]
```diff
@@ -82,13 +83,12 @@ that ACRN offers:
...
+  Other VMs, including the Service VM, may not even be aware of a pre-launched
+  VM's existence. The configuration of pre-launched VMs is static and must be
+  defined at build time. They are well-suited for safety-critical applications
+  and where very strict isolation, including from the Service VM, is desirable.
 
 * **Service VM**: A special VM, required for scenarios that have post-launched
   User VMs. The Service VM can access hardware resources directly by running
```
Comment (dbkinder) [inline doc/getting-started/overview_dev.rst:94]: Can you check the indenting of the rest of this paragraph.  Something looks wonky here.

File: doc/getting-started/overview_dev.rst [hunk 3]
```diff
@@ -116,29 +116,27 @@ meet their requirements without pre-launched VMs. Even if your application has
...
 
 * **Shared scenario:** This scenario represents a traditional computing, memory,
   and device resource sharing model among VMs. It has post-launched User VMs and
   the required Service VM. There are no pre-launched VMs in this scenario.
 
-* **Partitioned scenario:** This scenario has pre-launched User VMs to
-  demonstrate VM partitioning: the User VMs are independent and isolated, and
+* **Partitioned scenario:** This scenario has pre-launched User VMs only. It
+  demonstrates VM partitioning: the User VMs are independent and isolated, and
   they do not share resources. There is no need for the Service VM or Device
```
Comment (dbkinder) [inline doc/getting-started/overview_dev.rst:130]: It's worth adding that storage devices are not shared either, so each pre-launched VM requires its own boot device.

File: doc/getting-started/overview_dev.rst [hunk 4]
```diff
@@ -181,8 +179,8 @@ You must configure all of your target's BIOS settings before running the Board
 Inspector tool, because the tool records the current BIOS settings in the board
 configuration file.
 
-ACRN requires the BIOS settings listed in :ref:`gsg-board-setup` of the Getting
-Started Guide.
+ACRN requires the virtualization settings listed in :ref:`gsg-board-setup` of
```
Comment (dbkinder) [inline doc/getting-started/overview_dev.rst:182]: There are more than just virtualization settings in the BIOS that are needed, and I think we should mention BIOS here.
```suggestion
ACRN requires the BIOS settings listed in :ref:`gsg-board-setup` of
```

File: doc/getting-started/overview_dev.rst [hunk 5]
```diff
@@ -208,67 +206,23 @@ and :ref:`overview_dev_build`.
 |icon_host| Step 4: Generate a Scenario Configuration File and Launch Scripts
 *****************************************************************************
 
-The :ref:`acrn_configurator_tool` ``acrn_configurator.py`` enables you to
-configure your ACRN hypervisor and VMs via a web-based user interface on your
-development computer. Using the tool, you define your scenario settings and save
-them to a scenario configuration file. For scenarios with post-launched User
-VMs, you must also configure and generate launch scripts.
+The :ref:`acrn_configurator_tool`, found in the ACRN hypervisor source code,
```
Comment (dbkinder) [inline doc/getting-started/overview_dev.rst:209]: I believe we'll actually have a Debian package for the 3.0 release that will be used to install the configurator.
```suggestion
The :ref:`acrn_configurator_tool`
```

File: doc/getting-started/overview_dev.rst [hunk 6]
```diff
@@ -208,67 +206,23 @@ and :ref:`overview_dev_build`.
...
+enables you to configure your scenario settings via a graphical user interface
```
Comment (dbkinder) [inline doc/getting-started/overview_dev.rst:210]: Less passive would be:
```suggestion
lets you configure your scenario settings via a graphical user interface
```

File: doc/getting-started/overview_dev.rst [hunk 7]
```diff
@@ -208,67 +206,23 @@ and :ref:`overview_dev_build`.
...
-existing one, such as a predefined scenario described in
-:ref:`overview_dev_hw_scenario`. The tool's GUI enables you to edit the
-configurable items in the file, such as adding VMs, modifying VM attributes, or
-deleting VMs. The tool validates your inputs against your board configuration
-file. After validation is successful, the tool generates your custom scenario
-configuration file in XML format.
-
-Generate Launch Scripts
-=======================
+The tool guides you to import the board configuration file that you generated in
```
Comment (dbkinder) [inline doc/getting-started/overview_dev.rst:213]: Less passive would be:
```suggestion
The tool imports the board configuration file that you generated in
```

**Review by dbkinder — CHANGES_REQUESTED** [review]
A few comments/tweaks

File: doc/getting-started/overview_dev.rst (hunk 1 above)
Comment (gvancuts) [reply doc/getting-started/overview_dev.rst:87]: That's mixing singular and plural, isn't it? "a ... cores"

**Pull request merged**